src/xdocs/index.xml.vm

```diff
@@ -97,8 +97,25 @@
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
       <p>
-        Checkstyle is not using Semantic Versioning
-        (<a href="https://github.com/checkstyle/checkstyle/issues/3709">reasons in issue #3709</a>)
+        Checkstyle is following
+        <a href="https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/">
+          Hybrid Romantic and Semantic Versioning</a>:
```

Comment:
`Checkstyle is following Hybrid Romantic and Semantic Versioning` => `Checkstyle is following a hybrid of Romantic and Semantic Versioning`
(pure text version)

Reply:
I tried to copy name as is from linked article.

Reply:
Then I am ok to keep this as is.